transitions/index.js

```diff
@@ -135,20 +144,22 @@ const loadTransitions = (autoEnableSystemTransitions = true) => {
       return logger.warn(`transition ${key} not available.`);
     }
   });
+
+  if (loadError) {
+    logger.error(`Transitions are disabled until the above configuration errors are fixed.`);
```

Comment:
They really should fix the warn parts - it means they have configuration that they think is doing something and it's not. At best it's a failed attempt at loading an unreleased transition, at worst it's a typo in a transition name and they're not running a transition they think they are.

I've upgraded it to an error so sentinel transitions won't load if an unknown name is configured.